packages/react-instantsearch-hooks/package.json

```diff
@@ -42,10 +42,10 @@
     "@babel/runtime": "^7.1.2",
     "algoliasearch-helper": "^3.6.2",
     "dequal": "^2.0.0",
-    "instantsearch.js": "^4.32.0"
+    "instantsearch.js": "https://pkg.csb.dev/algolia/instantsearch.js/commit/3f954126/instantsearch.js"
```

Comment:
We'll be able to use the official InstantSearch.js dependency once algolia/instantsearch.js#4967 is released.

Reply:
Updated in 5fdac8d6eda991bab5fa3e159efad62ac7a8ecd4.